client/src/components/WhyIRMA/WhyIRMA.tsx

```diff
@@ -51,7 +55,7 @@ const IRMALogo = styled.img.attrs({ alt: 'IRMA Logo' })`
     margin: ${themeSpacing(1)} 0 ${themeSpacing(4)} ${themeSpacing(6)};
 `;
 
-const StyledLinkWithChevron = styled(AscLocal.LinkWithChevron).attrs({ forwardedAs: 'span' })`
+const StyledLinkWithChevron = styled(AscLocal.LinkWithChevron).attrs({ forwardedAs: 'h3' })`
```

Comment:
Doet dit iets? Ik zie nog steeds geen h3 in de DOM

Reply:
Die forwarded as zorgt dat dit component in de DOM wordt gerenderd als een h3. Als je het element inspecteert zie je dat het parent component van die span wel degelijk een h3 is. Maar hoe dan ook, ik weet eerlijk gezegd niet waarom die forwardedAs erin stond want het moet gewoon een link zijn. Pas het ff aan.